### PR TITLE
Dread: Final Dairon revision

### PR DESCRIPTION
**Pull request by Hugoshido**
Final review of Dairon

- renamed events
- added Grapple Beam movement (intermediate) to "Transport to Artaria" Area
- renamed "full samus controll" node to "Start Point"

### CODE REVIEW

File: randovania/games/dread/json_data/Dairon.txt [hunk 1]
```diff
@@ -278,7 +278,7 @@ Extra - asset_id: collision_camera_003
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Elevator to Ferenia - East Transport to Darion
       After Dairon - Pull Grapple Block below Ferenia Elevator
-  > Event - Pull Grapple Block
+  > Event - Pull Grapple Block below Elevator to Ferenia
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:281]: The event nodes don't have to match the resource name precisely. (if that were the case, there'd be a rule like we do for docks).
_Resources_ have to be unique and thus include the room name, but a shorter node name reads better.

Whenever Randovania displays a node, it takes care to include the area (and even world) names.

File: randovania/games/dread/json_data/Dairon.txt [hunk 2]
```diff
@@ -320,7 +320,7 @@ Extra - asset_id: collision_camera_004
               All of the following:
                   Flash Shift
                   Lay Cross Comb or Use Spin Boost
-  > Event - Blob (db_reg_b1_005)
+  > Event - Big Hub Blob
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:323]: Just Blob works.

File: randovania/games/dread/json_data/Dairon.txt [hunk 3]
```diff
@@ -363,17 +363,17 @@ Extra - asset_id: collision_camera_004
...
   * Layers: default
-  * Event Dairon - db_reg_b1_005
+  * Event Dairon - Big Hub Blob
   * Extra - actor_name: db_reg_b1_005
   * Extra - actor_def: actordef:actors/props/db_reg_b1_004/charclasses/db_reg_b1_004.bmsad
   > Door to Wide Beam Room
       Trivial
 
-> Event - Pull Grapple Block (grapplepulloff1x2_007); Heals? False
+> Event - Big Hub Grapple Block; Heals? False
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:374]: Just Grapple Block

File: randovania/games/dread/json_data/Dairon.txt [hunk 4]
```diff
@@ -640,7 +640,7 @@ Extra - asset_id: collision_camera_008
       Trivial
   > Tile Group (POWERBEAM) 2
       Trivial
-  > Event - Blob through wall (db_dif_b1_001)
+  > Event - Teleport to Cataris Blob ,through wall
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:643]: wrong comma place

File: randovania/games/dread/json_data/Dairon.txt [hunk 5]
```diff
@@ -671,9 +671,9 @@ Extra - asset_id: collision_camera_008
   > Door to Power Switch 1
       After Dairon - Power Switch 1
 
-> Event - Blob from shaft (db_dif_b1_001); Heals? False
+> Event - Teleport to Cataris Blob, from teleporter; Heals? False
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:674]: "Blob, from teleporter"

File: randovania/games/dread/json_data/Dairon.txt [hunk 6]
```diff
@@ -978,21 +978,22 @@ Extra - asset_id: collision_camera_011
...
+                  Grapple Beam and Grapple Movement (Intermediate)
           Any of the following:
               # Varia check
               Varia Suit
               All of the following:
                   # heat runs
                   Heat/Cold Runs (Intermediate) and Heat Damage ≥ 300
 
-> Event - Pull Grapple Block (grapplepulloff1x2_006); Heals? False
+> Event - Transport to Artaria Grapple Block; Heals? False
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:991]: Just Grapple Block

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

**Review by XenoWars — CHANGES_REQUESTED** [review]
Full Samus Control node can probably just straight up be deleted, as Start Point is also incorrect. Constant damage being default makes coming from the elevator a non-issue now. The elevator is also the start point of this room.

**Comment by XenoWars** [comment]
Samus Control removal looks good

File: randovania/games/dread/json_data/Dairon.txt [hunk 7]
```diff
@@ -320,7 +320,7 @@ Extra - asset_id: collision_camera_004
               All of the following:
                   Flash Shift
                   Lay Cross Comb or Use Spin Boost
-  > Event - Blob (db_reg_b1_005)
+  > Event -  Blob
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:323]: Blank space before Blob

File: randovania/games/dread/json_data/Dairon.txt [hunk 8]
```diff
@@ -671,9 +671,9 @@ Extra - asset_id: collision_camera_008
   > Door to Power Switch 1
       After Dairon - Power Switch 1
 
-> Event - Blob from shaft (db_dif_b1_001); Heals? False
+> Event -vBlob, from teleporter; Heals? False
```
Comment (henriquegemignani) [inline randovania/games/dread/json_data/Dairon.txt:674]: stray v

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**